crates/core/Cargo.toml

```diff
@@ -8,10 +8,13 @@ colored_json = "5.0.0"
 derive-getters = "0.3.0"
 derive_builder = "0.20.0"
 derive_more.workspace = true
+enquote = "1.1.0"
```

Comment:
Check this crate also... https://docs.rs/escape8259/0.5.3/escape8259/fn.escape.html